# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini
# Commit: 2f48a6566514
# Full Hash: 2f48a65665147ef98c0f1e7d26a99f1931f9470d
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-12 21:17:53
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r fc9333c3b9f7 -r 2f48a6566514 testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini
--- a/testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini	Tue Sep 12 11:25:16 2023 +0000
+++ b/testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini	Tue Sep 12 11:45:26 2023 +0000
@@ -1,8 +1,3 @@
 [SVGLength-ic.html]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-  [ic unit in SVGLength]
-    expected: FAIL
-
-  [Convert back to ic from new user unit value]
-    expected: FAIL